frontend/components/TodayInHistory.tsx

```diff
@@ -1,33 +1,15 @@
-import axiosWithoutAuth from "@/axiosWithoutAuth";
 import { ModuleUnion, Topic } from "@/types/modules";
-import { Card, CardContent, CardHeader, Grid, Skeleton } from "@mui/material";
-import axios from "axios";
+import { Grid } from "@mui/material";
 import Link from "next/link";
-import { FC, useEffect, useState } from "react";
+import { FC } from "react";
 import ModuleUnionCard from "./cards/ModuleUnionCard";
 
-const TodayInHistory: FC = () => {
-  const [items, setItems] = useState<Exclude<ModuleUnion, Topic>[]>([]);
-  const [loading, setLoading] = useState(true);
-
-  useEffect(() => {
-    const cancelTokenSource = axios.CancelToken.source();
-    axiosWithoutAuth
-      .get("/api/home/today_in_history/", { cancelToken: cancelTokenSource.token })
-      .then((response) => {
-        setItems(response.data);
-        setLoading(false);
-      })
-      .catch((error) => {
-        if (axios.isCancel(error)) return;
-        console.error(error);
-        setLoading(false);
-      });
-    return () => {
-      cancelTokenSource.cancel("component unmounted");
-    };
-  }, [loading]);
+export interface TodayInHistoryProps {
+  todayinhistoryData: Exclude<ModuleUnion, Topic>[];
```

Comment:
I'd like to rename this from `todayinhistoryData` to `items`.

Rationale:
- `todayinhistory` part is redundant since this is a param name for the `TodayInHistory` component
- Since it's an array, a plural variable name like `items` is more appropriate than `data` (which usually implies an object with keys)

Reply:
Changed todayinhistoryData -> todayModules. Is it better to stick to this one or use "items"?